local_init.bash

```diff
@@ -4,4 +4,9 @@ mkdir -p ~/.enigma
 mkdir -p ../build
 docker cp $CONTAINER_ID:/root/build/contracts ../build
 sed -e 's_http://contract_http://localhost_g;s_http://bootstrap_http://localhost_g' ../build/contracts/addresses.json > ../build/contracts/addresses.new
-mv ../build/contracts/addresses.new ../build/contracts/addresses.json
\ No newline at end of file
+mv ../build/contracts/addresses.new ../build/contracts/addresses.json
+
+echo "SGX_MODE=SW" > .env
```

Comment:
This is not straightforward. It should only create `.env` if it does not exist, because if it exists with another value it will overwrite

Reply:
Done